fronts-client/src/constants/feastPalettes.ts

```diff
@@ -502,49 +562,49 @@ export const feastCollectionPalettes: PaletteOption[] = [
 		imageURL: 'https://uploads.guim.co.uk/2024/03/21/Picnic.png',
 	},
 	{
-		id: 'potatos',
-		name: 'Potatos',
+		id: 'pizza',
+		name: 'Pizza',
 		palettes: {
 			light: {
-				foregroundHex: '#99614B',
+				foregroundHex: '#80191A',
 				backgroundHex: '#F9F9F5',
 			},
 			dark: {
-				foregroundHex: '#F0C5B4',
+				foregroundHex: '#FDF1F0',
 				backgroundHex: '#363632',
 			},
 		},
-		imageURL: 'https://uploads.guim.co.uk/2024/03/21/Potatos.png',
+		imageURL: 'https://uploads.guim.co.uk/2024/10/18/Pizza.png',
 	},
 	{
-		id: 'rice',
-		name: 'Rice',
+		id: 'potatos',
+		name: 'Potatos',
```

Comment:
Apparently should be `Potatoes` (I had to look this up! 😁 https://www.dictionary.com/browse/potato)